vue/wolox-books/src/views/Register.vue

```diff
@@ -0,0 +1,162 @@
+<template lang='pug'>
+  .register-container
+    img.wolox-icon(alt='Vue logo' src='../assets/wolox_logo.svg')
+    form.form-container(@submit.prevent="onSubmit")
+      p.title-form
+        |{{labels.title}}
+      .input-text-container(v-for='(field, index) in fields')
```

Comment:
remove index if you are not going to use it

Reply:
Sorry i forget put binding key with index. :key='index'